architecture/additional_concepts/sdn.adoc

```diff
@@ -122,3 +124,97 @@ so that the master does not schedule containers on it.
 Both options are presented as part of a practical use-case in the documentation
 for configuring link:../../admin_guide/routing_from_edge_lb.html[routing from an
 edge load-balancer to containers within OpenShift SDN].
+
+=== multitenant Plugin Operation
+
+The multitenant plugin isolates pods based on OpenShift projects.  Each project
+receives a unique Virtual Network ID (VNID) that identifies traffic from pods
+assigned to the project.  Pods from different projects cannot send packets to
+or receive packets from pods of a different project.
+
+However, the "default" project (which receives VNID 0) is more privileged in
+that it is allowed to communicate with all other pods, and all other pods can
+communicate with the "default" project.  This facilitates certain services
+(like the load balancer) that are run in containers themselves and must talk
+to all other containers.
+
+When OpenShift SDN is run on a node with the multitenant plugin, it configures
+five network devices:
```

Comment:
OK thanks. Reason I ask is cuz it says five above for the ovssubnet list as well; should that be four then?